data/seeds/011-mentor_intakes.js

```diff
@@ -7,8 +7,8 @@ const dummyData = [
     last_name: 'Jeopardy',
     current_comp: 'Amazin',
     tech_stack: 'Node.js, Axios',
```

Comment:
Pls update tech_stack to our current categories of frontend, backend, etc.

Reply:
Suggested changes are now implemented.